cmd/kubeadm/app/features/features.go

```diff
@@ -42,7 +42,10 @@ const (
 
 // InitFeatureGates are the default feature gates for the init command
 var InitFeatureGates = FeatureList{
-	PublicKeysECDSA:      {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
+	PublicKeysECDSA: {
+		FeatureSpec:        featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Deprecated},
+		DeprecationMessage: "The PublicKeysECDSA feature gate is deprecated and will be removed after the feature 'ClusterConfiguration.EncryptionAlgorithm' is added.",
```

Comment:
okay, 'ClusterConfiguration.EncryptionAlgorithm' is added right after the merge of this pr, might be better to say exactly release cycle instead.

Reply:
https://github.com/kubernetes/kubernetes/pull/120417#discussion_r1319364251